test/res.cookie.js

```diff
@@ -150,6 +150,31 @@ describe('res', function(){
       })
     })
 
+    describe('signed without secret', function(){
+      it('should throw an error', function(done){
+        var app = express();
+
+        app.use(cookieParser());
+
+        app.use(function(req, res){
+          try {
+            res.cookie('name', 'tobi', { signed: true });
+          } catch (err) {
+            return res.json({ status: 'OK' });
+          }
+
+          res.json({ status: 'FAIL' });
+        });
+
+        request(app)
+        .get('/')
+        .end(function(err, res){
```

Comment:
I think with supertest, this end block can simply be

    .expect({ status: 'OK' })